src/Commands/Env/ViewCommand.php

```diff
@@ -49,24 +53,8 @@ public function view($site_env, $options = ['print' => false,])
             return $url;
         }
 
-        // Otherwise attempt to launch it.
-        $cmd = '';
-        switch (php_uname('s')) {
-            case 'Linux':
-                $cmd = 'xdg-open';
-                break;
-            case 'Darwin':
-                $cmd = 'open';
-                break;
-            case 'Windows NT':
-                $cmd = 'start';
-                break;
-        }
-        if (!$cmd) {
-            throw new TerminusException("Terminus is unable to open a browser on this OS");
-        }
-        $command = sprintf('%s %s', $cmd, $url);
-        exec($command);
+        $this->getContainer()->get(LocalMachineHelper::class)->openUrl($url);
```

Comment:
Ok, I did some exploring and I'm not sure how easy it will be to use the Symfony helper system. Our commands aren't Symfony commands and our Models/Collections don't automatically have access to the Symfony application or components (nor should they). It will take some dependency gymnastics to make these helpers available to any class that might need them. We already have a mechanism in place to make use of the Leage DI container so I think we should continue to use that.

If this helper becomes useful enough we could promote it to it's own `LocalMachineHelperAwareInterface` but I'm not sure we're there quite yet.